prosody/rootfs/defaults/conf.d/jitsi-meet.cfg.lua

```diff
@@ -26,11 +26,20 @@ asap_accepted_issuers = { "{{ join "\",\"" (splitList "," .Env.JWT_ACCEPTED_ISSU
 asap_accepted_audiences = { "{{ join "\",\"" (splitList "," .Env.JWT_ACCEPTED_AUDIENCES) }}" }
 {{ end }}
 
-{{ if $ENABLE_XMPP_WEBSOCKET }}
+consider_bosh_secure = true;
+
 -- Deprecated in 0.12
 -- https://github.com/bjc/prosody/commit/26542811eafd9c708a130272d7b7de77b92712de
-cross_domain_websocket = { "{{ $PUBLIC_URL }}" };
-consider_bosh_secure = true;
+{{ $XMPP_CROSS_DOMAIN := $PUBLIC_URL }}
+{{ if eq (.Env.XMPP_CROSS_DOMAIN | default "false") "true"}}
+cross_domain_websocket = true
+cross_domain_bosh = true
+{{ else }}
+{{ if .Env.XMPP_CROSS_DOMAIN }}
```

Comment:
no default | to Bool ?

Reply:
We don't need a default or toBool because if a string is empty it's `false` by default.

Reply:
Ohh ok, you mean if someone wants to set `false` it breaks, right?

Reply:
Changed it.